### ISSUE TITLE
Coding style

### ISSUE THREAD

**Issue by MartinThoma** [TensorVision/TensorVision#2]
## Code
- [PEP 8](https://www.python.org/dev/peps/pep-0008/)
- [PEP 257](https://www.python.org/dev/peps/pep-0257/)
## Commits
- [How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/)
## Docstrings

Options: 
-  [Numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)
- [Google](http://google.github.io/styleguide/pyguide.html#Comments)

**Comment by MartinThoma** [comment]
@MarvinTeichmann Do you have any preference for docstrings? If not, then I would prefer the Numpydoc style.

**Comment by MarvinTeichmann** [comment]
@MartinThoma I am happy with both options. I guess Tensorflow uses natively the style Google? This might be a reason to choose this one.

**Comment by MartinThoma** [comment]
@MarvinTeichmann Yes, TensorFlow probably uses the Google style. However, I don't see how this is an argument for the Google Style.

I think the Numpydoc style is a little bit easier to read. (see http://bwanamarko.alwaysdata.net/napoleon/format_exception.html)

Lets see if https://www.quora.com/unanswered/Numpydoc-vs-Google-Docstring-style-What-arguments-are-there-for-which-style?srid=z393 gives an answer.

**Comment by MartinThoma** [comment]
A strong argument for the numpydoc style might be the possibility to add references.
